src/_components/CentreModal/CentreModal.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createOfficeLocation } from '../../_actions/officeLocations.actions';
+
+import CentreModalComponent from '../../components/CentreModal/CentreModal';
+
+class CentreModal extends Component {
```

Comment:
Please combine this component with the one in the `components/.../CentreModal.js` and remember that we decided to not add any component definition in a container file. Read more [here](https://github.com/AndelaOSP/art-dashboard/blob/develop/docs/best_practices/code_organization.md#_components-folder)